tests/unittests/unittest.cpp

```diff
@@ -455,6 +494,34 @@ TEST_CASE("AssociativeContainer", "[basics]") {
   cMap[clu3] = 42;
 
   REQUIRE(cMap[clu3] == 42);
+
+  // unordered associative containers
+
+  std::unordered_set<ExampleCluster> cUnorderedSet;
+  cUnorderedSet.insert(clu1);
+  cUnorderedSet.insert(clu2);
+  cUnorderedSet.insert(clu3);
+  cUnorderedSet.insert(clu4);
+  cUnorderedSet.insert(clu5);
+  cUnorderedSet.insert(clu1);
+  cUnorderedSet.insert(clu2);
+  cUnorderedSet.insert(clu3);
+  cUnorderedSet.insert(clu4);
+  cUnorderedSet.insert(clu5);
+
+  REQUIRE(cUnorderedSet.size() == 5);
```

Comment:
```suggestion
  REQUIRE(cUnorderedSet.size() == 5);

```
Minor nit to make association of assertion and setup easier to see.